manual/doctools/src/main/java/org/neo4j/doc/SettingsDescription.java

```diff
@@ -76,6 +78,11 @@ public static SettingsDescription describe( Class<?> settingClass )
                 {
                     Object rawDefault = setting.apply( from -> null );
                     defaultValue = rawDefault != null ? rawDefault.toString() : null;
+                    // Kludge--because for DURATION settings, the internal representation (MS) is leaked as the default value
+                    Optional<? extends Function<String, ?>> parser = setting.getParser();
+                    if (null != defaultValue && parser.isPresent() && Settings.DURATION.equals(parser.get())) {
+                        defaultValue = Long.toString(TimeUtil.DEFAULT_TIME_UNIT.convert(Long.valueOf(defaultValue), TimeUnit.MILLISECONDS));
```

Comment:
there's no immediate way to get the unit from TimeUtil. it's a good-to-have so let's defer it to the new tool in 3.2